### PR TITLE
Renamed TweetModel for better understanding and usage with `rejected` branch on SCE

### PR DESCRIPTION
**Pull request by T-Damer**

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ void (async () => {
   console.log('Checked post contracts')
 
   console.log('Starting contract listeners...')
-  await startContractListeners()
+  startContractListeners()
```
Comment (smyzin) [inline src/app.ts:30]: Why did you remove `await` from here?

**Review by smyzin — APPROVED** [review]
lgtm

File: src/app.ts (hunk 1 above)
Comment (T-Damer) [reply src/app.ts:30]: We don't need, the function is not asynchronous

**Review by backmeupplz — APPROVED** [review]
